src/biokbase/narrative/tests/test_appmanager.py

```diff
@@ -39,31 +41,102 @@ def get_method(tag, app_id, live=False):
     )
 
 
+def get_bulk_run_good_inputs():
```

Comment:
Changed this from a class variable to a get method because it's fundamentally meant to be mutated now